manifests/config/lookup_tables.pp

```diff
@@ -0,0 +1,10 @@
+class rsyslog::config::lookup_tables {
+  $::rsyslog::config::lookup_tables.each |$table, $config| {
+    rsyslog::component::lookup_table { $table:
+      * => {
```

Comment:
Ditto here - the '*'  notation serves no purpose here and is confusing

Reply:
No - you're right - sorry - I didn't spot the `+ $config` - the way you have done this is fine.... I just need more coffee